### DISCUSSION TITLE
Quest 3 Load

### DISCUSSION THREAD

**Discussion by popohoma** [praydog/REFramework#1005] [General Questions]
Hi Praydog,

Been hearing your famable name lately via lot of channels.  I am interested to have VR Mod for my purchased Street Fighter 6 and upcoming Dragon's Dogma 2 into my Quest 3 VR headset.  I am keen to find out the exact procedure so that I don't end up trial and error.  How would I do after I have downloaded your RE version? Should I copy it to each of my local game folder in PC (e.g. Street Fighter 6), and then copy it over to my Quest ? It will automatically detected as a game in Quest then?

Thanks,

Allen

**Comment by Satan-friend** [comment]
> Hi Praydog,
> 
> Been hearing your famable name lately via lot of channels. I am interested to have VR Mod for my purchased Street Fighter 6 and upcoming Dragon's Dogma 2 into my Quest 3 VR headset. I am keen to find out the exact procedure so that I don't end up trial and error. How would I do after I have downloaded your RE version? Should I copy it to each of my local game folder in PC (e.g. Street Fighter 6), and then copy it over to my Quest ? It will automatically detected as a game in Quest then?
> 
> Thanks,
> 
> Allen

Hello, I've got Quest 3 too, just put the contents of the REFramework into the game folder. Then on Quest download Steam Link, open it and run the game in SteamVR. The REFramework works great.

*If the connection is unstable (lags) then make sure you have a Wifi 6 router or newer. I myself had problems with Wifi 5, but after changing to a TP-Link AX3000 the streaming works perfectly.

**Reply by popohoma** [reply]
Thank you for your reply Praydog.  I will try it out tonight.  Since you have Quest 3, are you able to play Street Fighter 6 by PS4 or Microsoft Controllers set up in Quest ? I don't think I am able to use P4 controller via Steam link / Steam VR app.
